### PR TITLE
docs(pagination): Add demo for content switching

### LINKED ISSUE #3771: docs(pagination): Add demo for content switching
related to #3237

### PR DESCRIPTION
**Pull request by artolshansky**
When you click on next page or page number - content will be switched 

close #3771

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated demos.

### CODE REVIEW

File: demo/src/app/components/+pagination/demos/content-switching/content-switching.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'demo-pagination-content-switching',
+  templateUrl: './content-switching.html'
+})
+export class DemoPaginationContentSwitchingComponent implements OnInit {
+  contentArray = Array.apply(null, Array(88)).map((v: any, i: number) => `Content line ${i + 1}`);
```
Comment (IlyaSurmay) [inline demo/src/app/components/+pagination/demos/content-switching/content-switching.ts]: fill the array in the constructor

File: demo/src/app/components/+pagination/demos/content-switching/content-switching.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  returnedArray: string[];
+
+  ngOnInit(): void {
+    if (!this.returnedArray){
```
Comment (IlyaSurmay) [inline demo/src/app/components/+pagination/demos/content-switching/content-switching.ts]: why do you need this check if `returnedArray` is undefined by default?

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

File: demo/src/app/components/+pagination/demos/content-switching/content-switching.ts (hunk 2 above)
Comment (olshansky) [reply demo/src/app/components/+pagination/demos/content-switching/content-switching.ts]: @IlyaSurmay just habit

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Please, move content displaying to display it under pagination and add border to it
contenttop

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

File: demo/src/app/components/+pagination/demos/content-switching/content-switching.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<div class="row">
+  <div class="col-xs-12 col-12">
+    <div class="content-wrapper" style="border: 1px solid #ddd; border-radius: 4px; padding-left: 10px; margin-bottom: 10px">
```
Comment (valorkin) [inline demo/src/app/components/+pagination/demos/content-switching/content-switching.html]: no inline styles

File: demo/src/app/components/+pagination/demos/content-switching/content-switching.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+@Component({
+  selector: 'demo-pagination-content-switching',
+  templateUrl: './content-switching.html'
+})
+export class DemoPaginationContentSwitchingComponent implements OnInit {
+  contentArray = new Array(90).fill('');
+  returnedArray: string[];
+
+  ngOnInit(): void {
+    this.contentArray = this.contentArray.map((v: any, i: number) => `Content line ${i + 1}`);
```
Comment (valorkin) [inline demo/src/app/components/+pagination/demos/content-switching/content-switching.ts]: no `any`

File: demo/src/app/components/+pagination/demos/content-switching/content-switching.ts [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    this.returnedArray = this.contentArray.slice(0, 10);
+  }
+
+  pageChanged(event: any): void {
```
Comment (valorkin) [inline demo/src/app/components/+pagination/demos/content-switching/content-switching.ts]: event type

**Review by valorkin — CHANGES_REQUESTED** [review]

**Review by valorkin — APPROVED** [review]

**Pull request merged**